code/edge/src/test/java/com/adobe/marketing/mobile/EdgeStateTests.java

```diff
@@ -44,7 +44,7 @@ public void setUp() {
 		mockSharedStateCallback =
 			new EdgeSharedStateCallback() {
 				@Override
-				public Map<String, Object> getSharedState(final String stateOwner, final Event event) {
+				public SharedStateResult getSharedState(final String stateOwner, final Event event) {
```

Comment:
The changes in EdgeStateTests were done automatically when I made the API changes. The test classes will be updated in another PR.